kep/9-capacity-scheduling/README.md

```diff
@@ -262,6 +262,37 @@ guaranteed and hence non-preemptable.
 In the end, UserA consumes 4 GPUs and UserB consumes 6 GPUs, satisfying each 
 user's "min" quota.
 
+#### Story 2
+We assume three elastic quotas are defined: QuotaA (min:3,  max:4)、 QuotaB
+(min:4, max:6) and QuotaC (min:3,  max:4), the quota unit is the number of GPUs. UserA in blue color
+consumes QuotaA,UserB in red color consumes QuotaB and UserC in green color consumes QuotaC. The entire cluster
+has 10 GPUs available hence the sum of quota min is equal to the cluster capacity.
+
+Initially, UserA consumes 2 GPUs,UserB consumes 2 GPUs and UserC consumes 3 GPUS. All consumptions
+are within the reserved "min" quota range.
+
+![5](./5.png)
+
+Next,UserA consumes 4 GPUs reaching the "max" quota of QuotaA.UserB consumes 3 GPUs.UserC still consumes 3 GPUs.
+
+![6](./6.png)
+
+Later,UserB submits a pod that requests 1 GPU, at this time the cluster does not have enough resources and needs 
```

Comment:
we should keep the same expression `pod` or `task`

Reply:
done